### PR TITLE
Fixed Experience Gain message

### PR DESCRIPTION
**Pull request by dastgirp**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Since macro is used in message, I believe we can't move it to messages.conf yet, and it will display the format specifier instead of value

**Issues addressed:** 
None

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: conf/messages.conf [hunk 1]
```diff
@@ -694,7 +694,7 @@
 886: Gained %dz.
 887: %s stole an Unknown Item (id: %i).
 888: %s stole %s.
-889: Experience Gained Base:%"PRIu64" (%.2f%%) Job:%"PRIu64" (%.2f%%)
```
Comment (4144) [inline conf/messages.conf]: probably better left format string as is. but in C code at first replace ``%"PRIu64"`` to actual format parameters. then use fixed format string in sprintf?

Comment (dastgirp) [reply conf/messages.conf]: oh, I didn't knew. I always though PRIu64 corresponds to different identifier based on OS type (without checking it). 
It seems to be same identifier for every OS. changed it.

Comment (MishimaHaruna) [reply conf/messages.conf]: This seems to be a Microsoft extension, not compatible with other compiles (at least not compatible with gcc-6 on debian, gcc-9 on gentoo and gcc-9 on macos)

```
#include <stdint.h>
#include <stdio.h>

int main(void)
{
	int64_t foo = 1234567890123456789ull;
	printf("%I64u\n", foo);
}
```



```
debian$ gcc-6 -std=c11 -pedantic -Wall -Wextra test_I64u.c && ./a.out
test_I64u.c: In function ‘main’:
test_I64u.c:7:9: warning: ISO C does not support the 'I' printf flag [-Wformat=]
  printf("%I64u\n", foo);
         ^~~~~~~~~
test_I64u.c:7:14: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 2 has type ‘int64_t {aka long int}’ [-Wformat=]
  printf("%I64u\n", foo);
              ^
                                                      2112454933
```

```
gentoo % gcc-9.2.0 -std=c11 -pedantic -Wall -Wextra test_I64u.c && ./a.out
test_I64u.c: In function ‘main’:
test_I64u.c:7:9: warning: ISO C does not support the 'I' printf flag [-Wformat=]
    7 |  printf("%I64u\n", foo);
      |         ^~~~~~~~~
test_I64u.c:7:14: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 2 has type ‘int64_t’ {aka ‘long int’} [-Wformat=]
    7 |  printf("%I64u\n", foo);
      |          ~~~~^     ~~~
      |              |     |
      |              |     int64_t {aka long int}
      |              unsigned int
      |          %I64lu
                                                      2112454933
```

```
macos% gcc-9 -std=c11 -pedantic -Wall -Wextra test_I64u.c && ./a.out
test_I64u.c: In function 'main':
test_I64u.c:7:9: warning: ISO C does not support the 'I' printf flag [-Wformat=]
    7 |  printf("%I64u\n", foo);
      |         ^~~~~~~~~
test_I64u.c:7:14: warning: format '%u' expects argument of type 'unsigned int', but argument 2 has type 'int64_t' {aka 'long long int'} [-Wformat=]
    7 |  printf("%I64u\n", foo);
      |          ~~~~^     ~~~
      |              |     |
      |              |     int64_t {aka long long int}
      |              unsigned int
      |          %I64llu
test_I64u.c:4:14: warning: unused parameter 'argc' [-Wunused-parameter]
    4 | int main(int argc, char *argv[])
      |          ~~~~^~~~
test_I64u.c:4:26: warning: unused parameter 'argv' [-Wunused-parameter]
    4 | int main(int argc, char *argv[])
      |                    ~~~~~~^~~~~~
I64u
```

```
macos% clang -std=c11 -pedantic -Wall -Wextra test_I64u.c && ./a.out
test_I64u.c:7:11: warning: length modifier 'I64' results in undefined behavior or no effect with 'u' conversion specifier [-Wformat]
        printf("%I64u\n", foo);
                ~^~~~
test_I64u.c:4:14: warning: unused parameter 'argc' [-Wunused-parameter]
int main(int argc, char *argv[])
             ^
test_I64u.c:4:26: warning: unused parameter 'argv' [-Wunused-parameter]
int main(int argc, char *argv[])
                         ^
3 warnings generated.
I64u
```

I suggest to use `%llu`, and do an explicit cast to `(unsigned long long)` in the printf function that references this messages.

**Comment by bWolfie** [comment]
bump amigos

File: conf/messages.conf (hunk 1 above)
Comment (MishimaHaruna) [reply conf/messages.conf:697]: I applied the change I suggested and force-pushed

**Pull request merged**